nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/controller/service/StandardControllerServiceNode.java

```diff
@@ -134,6 +135,7 @@ public StandardControllerServiceNode(final LoggableComponent<ControllerService>
         super(id, validationContextFactory, serviceProvider, componentType, componentCanonicalClass, reloadComponent, extensionManager, validationTrigger, isExtensionMissing);
         this.serviceProvider = serviceProvider;
         this.active = new AtomicBoolean();
+        this.moving = new AtomicBoolean(false);
```

Comment:
Adding this temporary state to the Node object raises a number of questions about the implementation, as it does not seem to fit with the other stateful properties.